src/applications/check-in/day-of/containers/withLoadedData.jsx

```diff
@@ -1,134 +0,0 @@
-import React, { useCallback, useEffect, useState, useMemo } from 'react';
-import PropTypes from 'prop-types';
-import { useDispatch, batch, useSelector } from 'react-redux';
-import { api } from '../../api';
-import { useSessionStorage } from '../../hooks/useSessionStorage';
-import { URLS } from '../../utils/navigation/day-of';
-import { useFormRouting } from '../../hooks/useFormRouting';
-import {
-  receivedEmergencyContact,
-  receivedDemographicsData,
-  receivedNextOfKinData,
-  receivedMultipleAppointmentDetails,
-  triggerRefresh,
-  receivedDemographicsStatus,
-} from '../../actions/day-of';
-import { focusElement } from 'platform/utilities/ui';
-
-import { makeSelectCheckInData } from '../hooks/selectors';
-
-const withLoadedData = Component => {
-  const Wrapped = ({ ...props }) => {
-    const [isLoading, setIsLoading] = useState();
-    const { isSessionLoading, router } = props;
-    const { goToErrorPage } = useFormRouting(router, URLS);
-    const selectCheckInData = useMemo(makeSelectCheckInData, []);
-    const checkInData = useSelector(selectCheckInData);
-    const { getCurrentToken } = useSessionStorage(false);
-    const {
-      context,
-      appointments,
-      demographics,
-      nextOfKin,
-      demographicsStatus,
-      emergencyContact,
-    } = checkInData;
-
-    const dispatch = useDispatch();
-    const setSessionData = useCallback(
-      (payload, token) => {
-        batch(() => {
-          const {
-            appointments: appts,
-            demographics: demo,
-            patientDemographicsStatus,
-          } = payload;
-          dispatch(triggerRefresh(false));
-          dispatch(receivedMultipleAppointmentDetails(appts, token));
-          dispatch(receivedDemographicsStatus(patientDemographicsStatus));
```

Comment:
Why was this action removed?

Reply:
I removed this because the form updater is handing everything to do with demographic statuses and we no longer need them to be in the store.